src/dmd/initsem.d

```diff
@@ -374,7 +374,8 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, Type t,
             sc = sc.endCTFE();
         if (i.exp.op == TOK.error)
         {
-            return new ErrorInitializer();
+            i.exp = new ErrorExp();
```

Comment:
There's something else wrong here. `.op` should NEVER be `TOK.error` while not being an `ErrorExp`. Need to find out where `.op` was set to `TOK.error` for a proper fix.

Reply:
Yeah but it IS  an error exp actually. I can return `i` directly after asserting that it is.